ebean-core/src/main/java/io/ebeaninternal/server/querydefn/DefaultOrmQuery.java

```diff
@@ -1298,8 +1300,10 @@ public int queryBindHash() {
   public HashQuery queryHash() {
     // calculateQueryPlanHash is called just after potential AutoTune tuning
     // so queryPlanHash is calculated well before this method is called
-    int hc = queryBindHash();
-    return new HashQuery(queryPlanKey, hc);
+    BindHash hash = bindHashAlgorithm == null ? new HashCodeBindHash() : new MdBindHash(bindHashAlgorithm);
```

Comment:
@rbygrave  
- Should I make the whole thing configurable via ServerConfig or can/should we use MD5 as default?
- Should I keep the old (dangerous) HashCodeBindHash implementation?